builtin/providers/aws/resource_aws_rds_cluster.go

```diff
@@ -121,13 +121,21 @@ func resourceAwsRDSCluster() *schema.Resource {
 
 			"master_username": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
+				Computed: true,
+				Optional: true,
 				ForceNew: true,
 			},
 
 			"master_password": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
+			},
+
+			"snapshot_identifier": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: false,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
```

Comment:
This isn't required - I know it's in the db_instance schema, but we only really use it when we are specifying a List